django_redshift_backend/base.py

```diff
@@ -36,6 +36,7 @@ class DatabaseFeatures(BasePGDatabaseFeatures):
     supports_column_check_constraints = False
     can_distinct_on_fields = False
     allows_group_by_selected_pks = False
+    has_native_uuid_field = False
```

Comment:
Memo: This class variable has been introduced at Django-1.8.0 https://github.com/django/django/commit/39d95fb6ada99c59d47fa0eae6d3128abafe2d58

With django 1.11, 2.0 and 2.1:
https://github.com/django/django/blob/838e432e3e5519c5383d12018e6c78f8ec7833c1/django/db/models/fields/__init__.py#L2317-L2319